test/test_move.py

```diff
@@ -165,3 +165,103 @@ def test_move_changes_workspace_layout(sway):
     assert ws.layout == 'splitv'
     assert ws.type == 'workspace'
     assert ws.name == '1'
+
+def test_in_and_out_of_vsplit(sway):
+    '''
+    (view1 focus) (container L_VERT (view2) (view3 focus-inactive))
+    -> move right
+    (container L_VERT (view2) (view3) (view1 focus))
+    -> move right
+    (container L_VERT (view2) (view3 focus-inactive)) (view1 focus)
+    -> move left
+    (container L_VERT (view2) (view3) (view1 focus))
+    '''
+    view1 = sway.open_window('view1')
+    view2 = sway.open_window('view2')
+    view2.command('splitv')
+    view3 = sway.open_window('view3')
+
+    view1.focus()
+    sway.ipc.command('move right')
+
+    tree = sway.ipc.get_tree()
+    ws = tree.workspaces()[0]
+    assert len(ws.nodes) == 1
+    vsplit = ws.nodes[0]
+    assert vsplit.nodes[0].id == view2.id
+    assert vsplit.nodes[1].id == view3.id
+    assert vsplit.nodes[2].id == view1.id
+    assert vsplit.nodes[2].focused
+
+    sway.ipc.command('move right')
+
+    tree = sway.ipc.get_tree()
+    ws = tree.workspaces()[0]
+    assert len(ws.nodes) == 2
+    assert len(ws.nodes[0].nodes) == 2
+    assert ws.nodes[0].nodes[0].id == view2.id
+    assert ws.nodes[0].nodes[1].id == view3.id
+    assert ws.nodes[1].id == view1.id
+    assert ws.nodes[1].focused
+
+    sway.ipc.command('move left')
+    tree = sway.ipc.get_tree()
+    ws = tree.workspaces()[0]
+    assert len(ws.nodes) == 1
+    vsplit = ws.nodes[0]
+    assert vsplit.nodes[0].id == view2.id
+    assert vsplit.nodes[1].id == view3.id
+    assert vsplit.nodes[2].id == view1.id
+    assert vsplit.nodes[2].focused
+
+def test_move_up_and_right(sway):
+    '''
+    ((container L_VERT (view1 focus) (view2) (view3)) (view4))
+    -> move up
+    -> move right
+    -> move left
+    ((container L_VERT (view2) (view3)) (view1 focus) (view4))
+    '''
+    view1 = sway.open_window('view1')
+    view1.command('splitv')
+    view2 = sway.open_window('view2')
+    view3 = sway.open_window('view3')
+    view4 = sway.open_window('view4')
+    sway.ipc.command('move right')
```

Comment:
Is this necessary? I don't see this on the test description.

Reply:
The description is meant to say: 1 horizontal container with one vertical container with 3 windows and one window.

Reply:
:+1: Missed the `()`.